api/routes.go

```diff
@@ -2,13 +2,15 @@ package api
 
 import (
 	"github.com/go-macaron/binding"
+	"github.com/go-macaron/gzip"
 	"github.com/raintank/metrictank/api/middleware"
 	"github.com/raintank/metrictank/api/models"
 	"gopkg.in/macaron.v1"
 )
 
 func (s *Server) RegisterRoutes() {
 	r := s.Macaron
+	r.Use(gzip.Gziper())
```

Comment:
can we put this behind a config flag? that would make it easier to compare the plain vs zipped experience. or are we super confident that gzip is better in all cases? (if so why)